content/dev-guide/coding-style.md

```diff
@@ -0,0 +1,184 @@
++++
+title = 'Coding style and conventions'
+date = 2023-10-27T13:42:48-07:00
+weight = 40
+draft = false
++++
+
```

Comment:
As an overall comment, I think these changes are too large to review in one PR. Generally, I think if we instead broke up this PR into "Dev guide", "Usage guide" and "Misc" PRs it would be way easier to review. Right now, I can tell that you've re-ordered some stuff, but it's really hard to tell what got put where because there's so many changed files.

Reply:
You're welcome to copy this over onto different branches.